runelite-client/src/main/java/net/runelite/client/plugins/party/PartyPingOverlay.java

```diff
@@ -44,12 +44,14 @@
 {
 	private final Client client;
 	private final PartyPlugin plugin;
+	private final PartyConfig config;
 
 	@Inject
-	private PartyPingOverlay(final Client client, final PartyPlugin plugin)
+	private PartyPingOverlay(final Client client, final PartyPlugin plugin, final PartyConfig config)
```

Comment:
Why does this suddenly need a config ref while nothing using the config is added